script/tool/lib/src/publish_plugin_command.dart

```diff
@@ -93,6 +101,9 @@ class PublishPluginCommand extends PluginCommand {
   static const String _remoteOption = 'remote';
   static const String _allChangedFlag = 'all-changed';
   static const String _dryRunFlag = 'dry-run';
+  static const String _yesOption = 'yes';
```

Comment:
How about `--skip-confirmation` since that makes it self-documenting where it's used?

(Also because `no-skip-confirmation` makes a lot more sense for the negated version than `no-yes`.)